lib/utils.js

```diff
@@ -56,9 +43,9 @@ utils.isPath = function isPath(pattern, opts) {
  * @return {Function}
  */
 
-utils.hasPath = function hasPath(pattern, opts) {
+utils.pathContains = function(pattern) {
   return function(fp) {
-    return utils.unixify(pattern, opts).indexOf(fp) !== -1;
```

Comment:
and yeah lol, it makes me a little nervous too. 

![image](https://cloud.githubusercontent.com/assets/383994/15325248/ce2f281e-1c17-11e6-99a5-dbdd99707f5f.png)